config_models/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.4'
+__version__ = '0.1.5'
```

Comment:
@jmbowman I think you're correct - it's not necessary in regards to the changes. However, I already tagged a previous commit hash as `0.1.4` and pushed it to GitHub before I saw that tests had failed. Too hasty! To be on the safe side, I decided to bump the version again.